product_docs/docs/pgd/4/rel_notes/pgd_4.3.1-1_rel_notes.mdx

```diff
@@ -0,0 +1,21 @@
+---
+title: "Release notes for EDB Postgres Distributed version 4.3.1-1"
+navTitle: "Version 4.3.1-1"
+---
+
+EDB Postgres Distributed version 4.3.1-1 is a minor release of EDB Postgres Distributed 4, which includes bug fixes for issues identified in previous versions.
+
+It includes HARP 2.2.3 with bug fixes and enhancements. If you are using HARP 2.2.3 or earlier, we recommend that you upgrade to HARP 2.3.0.
+
+!!! Note
+    This version is required for EDB Postgres Advanced Server versions 12.15, 13.11, 14.8, and later.
+
+
+### Updated 2023 July 12
```

Comment:
Remove this as the release date is mentioned on the index page. Also this is not present on page for 4.3.0-1